src/layouts/README.md

```diff
@@ -0,0 +1,5 @@
+# Layouts
+
+Layouts are reusable components that wrap around pages. They are used to provide a consistent look and feel across multiple pages.
```

Comment:
The word "wrap" here made me wonder what happens if the map wraps; turns out that if you go all the way around the world the coloured bit goes away! This sounds innocent enough but might create some visual weirdnesses and definitely calls for an eventual reset view button.  Better would be for the geojson to display at longiude 540 etc; there might be a leaflet option for this too. (another for later!)